distribution/ddf-common/src/main/resources/bin/ddf

```diff
@@ -60,12 +47,6 @@ shutdown_no_java_home_or_jre_home() {
 }
 
 attempt_startup() {
```

Comment:
Last time I worked in this area DDF did not provide good communication if it could not connect to Solr. To check, I would browse the UI. If I Intrigue didn't work right, I knew it was a Solr connection problem. Has the situation improved so that both admin and user can know right away if DDF is missing its connection to Solr?

Reply:
Patrick made improvements to the reliability of the Solr client factory.  I am not sure if its logging has addressed the communication issues if something goes wrong.  While working on this PR, I usually saw something from the zookeeper or Solr client if something was not working.

Reply:
You will definitely see messages from the Solr client in the logs if there is a communication problem. But unless you look in the logs, you might not know why the system is not working. It is probably outside the scope of your PR, but I recommend a follow-on PR that alerts admins and UI users that something is wrong if DDF cannot communicate with Solr.